src/signed_stubs.c

```diff
@@ -0,0 +1,143 @@
+#if !__USE_MINGW_ANSI_STDIO && (defined(__MINGW32__) || defined(__MINGW64__))
+#define __USE_MINGW_ANSI_STDIO 1
+#endif
+
+#include <caml/mlvalues.h>
+#include <caml/custom.h>
+#include <caml/alloc.h>
+#include <caml/intext.h>
+#include <caml/fail.h>
+
+#include <inttypes.h>
+#include <stdint.h>
+#include <limits.h>
+#include <stdio.h>
+
+#define OCAML_INTEGERS_INTERNAL 1
+#include "ocaml_integers.h"
+
+CAMLprim value integers_int_size(value unit)
```

Comment:
`Sys.int_size` was added in 4.03, so I copied this small function from the compiler distribution.

Reply:
I'd prefer to update the minimum requirement to OCaml 4.03 and call the function from the distribution